go.mod

```diff
@@ -4,6 +4,7 @@ go 1.16
 
 require (
 	github.com/golang/mock v1.6.0
+	github.com/golang/protobuf v1.5.3
```

Comment:
not sure why this got added back. Created https://github.com/theparanoids/crypki/pull/285 which doesn't have it. Can we merge that one instead @dmitris ?